scripts/config

```diff
@@ -47,10 +47,14 @@ get__data_multimedia() {
     fi
 }
 get__last_backups() {
-    cat << EOF
+    if [ "${old[state]}" == "repository uncreated" ]; then
```

Comment:
The `state` variable is indeed initialized with this value during the installation:
https://github.com/YunoHost-Apps/borg_ynh/blob/02913291eb95ad00ba155f59a3307a30637151c9/scripts/install#L32

But this is the only place this variable is set to this value and it's overwritten for example [in case of failure](https://github.com/YunoHost-Apps/borg_ynh/blob/02913291eb95ad00ba155f59a3307a30637151c9/conf/backup-with-borg#L76), even if no backups have been created.

I rather suggest to store the backup list in a variable and check whether its empty or not